src/test/kotlin/uk/gov/justice/digital/hmpps/pecs/jpc/integration/ManagePricesTest.kt

```diff
@@ -0,0 +1,92 @@
+package uk.gov.justice.digital.hmpps.pecs.jpc.integration
+
+import org.fluentlenium.core.annotation.Page
+import org.junit.jupiter.api.MethodOrderer.OrderAnnotation
+import org.junit.jupiter.api.Order
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestMethodOrder
+import uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages.AddPricePage
+import uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages.ChooseSupplierPage
+import uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages.DashboardPage
+import uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages.JourneysForReviewPage
+import uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages.ManageJourneyPricePage
+import uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages.SelectMonthYearPage
+import uk.gov.justice.digital.hmpps.pecs.jpc.integration.pages.UpdatePricePage
+import uk.gov.justice.digital.hmpps.pecs.jpc.price.Money
+import uk.gov.justice.digital.hmpps.pecs.jpc.price.Supplier
+import java.time.LocalDate
+import java.time.Month
+import java.time.Year
+
+@TestMethodOrder(OrderAnnotation::class)
+internal class ManagePricesTest : IntegrationTest() {
+
+  @Page
+  private lateinit var dashboardPage: DashboardPage
+
+  @Page
+  private lateinit var chooseSupplierPage: ChooseSupplierPage
+
+  @Page
+  private lateinit var selectMonthYearPage: SelectMonthYearPage
+
+  @Page
+  private lateinit var journeysForReviewPage: JourneysForReviewPage
+
+  @Page
+  private lateinit var addPricePage: AddPricePage
+
+  @Page
+  private lateinit var manageJourneyPricePage: ManageJourneyPricePage
+
+  @Page
+  private lateinit var updatePricePage: UpdatePricePage
+
+  @Test
+  @Order(1)
```

Comment:
Due to the fact these are integration tests (i.e. data is persisted across tests) the tests are being ordered so we can add then add then update the price for the same journey.

Reply:
Making tests run in a specific order is a bit of a smell, ideally they'd be completely independent. I would say it's perfectly fine to duplicate the add price bit in the update price test - it documents that a price has to have been added before it can be updated, so it acts as a documentation of how the system works.